src/middlewares/authorization.js

```diff
@@ -1,20 +1,19 @@
 import { verifyToken } from '../utils/authentication';
 import ResponseHandler from '../utils/ResponseHandler';
 
-// eslint-disable-next-line import/prefer-default-export
-export const checkToken = (req, res, next) => {
-  const token = req.headers['x-access-token']
-    || req.query.token
-    || req.headers.authorization;
-  if (token) {
+const checkToken = (req, res, next) => {
+  const queryToken = req.query.token;
+  const accessToken = req.headers['x-access-token'];
+  const bearerToken = req.headers.authorization;
+  if (queryToken || accessToken || bearerToken) {
+    const token = queryToken || accessToken || bearerToken.split(' ')[1];
```

Comment:
was it necessary to edit did lines 4-9?. was the original code that was there causing a bug in the app?. please, revert what you dod here.